env/includes/settings/tk-houdini.yml

```diff
@@ -70,7 +70,7 @@ settings.tk-houdini.project:
       location: "@apps.tk-multi-about.location"
     tk-multi-screeningroom: "@settings.tk-multi-screeningroom.rv"
     tk-multi-shotgunpanel: "@settings.tk-multi-shotgunpanel"
-    tk-multi-workfiles2: "@settings.tk-multi-workfiles2"
+    tk-multi-workfiles2: "@settings.tk-multi-workfiles2.houdini"
```

Comment:
Why do you need to add these suffixes everywhere?

Reply:
Adding the suffix `houdini` points the `tk-multi-workfiles2` settings to a different location.

Before, we were pointing to the settings at location `@settings.tk-multi-workfiles2` [defined here](https://github.com/shotgunsoftware/tk-config-default2/blob/2fb6c495cf1c4877945b50842e859911d55953e6/env/includes/settings/tk-multi-workfiles2.yml#L32), but since I needed to add the new hook specific to Houdini, I had to create a new settings location `@settings.tk-multi-workfiles2.houdini` that is just a copy of the old one but with the added hook. The new settings are defined [here](https://github.com/shotgunsoftware/tk-config-default2/blob/2fb6c495cf1c4877945b50842e859911d55953e6/env/includes/settings/tk-multi-workfiles2.yml#L172)